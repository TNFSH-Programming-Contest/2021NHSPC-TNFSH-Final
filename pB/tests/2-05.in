8 7 2
7 4 6 3 2 5
5 7 2 1 6 3
2 6 7 1 3 4
1 5 7 4 3 2
4 3 1 6 7 5
3 6 1 7 5 4
1 5 2 3 6 7
6 2 1 4 3 5
7 5 8 2 3 7 4 6
6 4 8 7 3 1 2
8 6 4 7 8 2 3 5 1
6 1 8 5 4 6 3
7 2 8 1 4 7 6 5
7 5 8 1 6 3 7 2
7 4 5 7 6 2 1 3
