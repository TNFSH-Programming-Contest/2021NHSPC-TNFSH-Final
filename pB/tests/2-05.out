2 4 7
1 3
1 6
1 5
1 2
1 8
1 1
